build-system/babel-config/pre-closure-config.js

```diff
@@ -72,8 +73,9 @@ function getPreClosureConfig() {
     './build-system/babel-plugins/babel-plugin-transform-amp-asserts',
     argv.esm ? filterImportsPlugin : null,
     // TODO(erwinm, #28698): fix this in fixit week
-    //argv.esm
-    //? './build-system/babel-plugins/babel-plugin-transform-function-declarations'
+    // argv.esm
+    //?
+    //'./build-system/babel-plugins/babel-plugin-transform-function-declarations'
```

Comment:
Maybe revert this, since it doesn't change anything?

Reply:
Done